v2.0/src/Components/Pages/ActionsPage/OneActionPage.js

```diff
@@ -37,22 +37,42 @@ class OneActionPage extends React.Component {
       showTestimonialLink: false,
       numberToShow: 3,
       tab: "description",
-      question: null
+      question: null,
+      action: null,
     };
     this.handleChange = this.handleChange.bind(this);
   }
+
   componentDidMount() {
     window.addEventListener("resize", this.chooseFontSize);
+
+    console.log("componentDidMount");
+    const { id } = this.props.match.params;
+    this.fetch(id)
+  }
+
+  componentDidUpdate(prevProps) {
```

Comment:
i am not really a fun of using these componentDidMount and componentDidUpdate stuff.
What's the goal here?

Reply:
Reading the doc: https://reactjs.org/docs/react-component.html#componentdidupdate
they suggest these two routines are ideal for network requests to load data.  The componentDidUpdate was needed since you can switch from one action to another on the list without the component being loaded.